packages/util/src/functions/polyfillDispose.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Polyfill for `Symbol.dispose` and `Symbol.asyncDispose` which is used as a part of
+ * {@link https://github.com/tc39/proposal-explicit-resource-management}. Node versions below 18.x
+ * don't have these symbols by default, so we need to polyfill them.
+ */
+export function polyfillDispose() {
```

Comment:
Isn't this wrong? If there's a different tool polyfilling this and then discord.js is imported, we overwrite the other polyfill with a completely different symbol - pain to debug.

Reply:
Discord.js wouldn't write polyfill in that situation because Symbol.dispose would already be defined.

Reply:
Oh sorry, I completely blanked on `??=`